src/components/Docs/SnippetViewer/SdkSetup.tsx

```diff
@@ -93,8 +93,8 @@ class MyProgram {
 ${importSdkStatement(lang, languageMappings)}
 
 configuration = openfga_sdk.Configuration(
-    scheme = os.environ.get('FGA_API_SCHEME'),
-    host = os.environ.get('FGA_API_HOST'),
+    api_scheme = os.environ.get('FGA_API_SCHEME'),
```

Comment:
Maybe we can add `FGA_API_SCHEME` above, considering we're using it here?

Reply:
I am not sure what do you mean